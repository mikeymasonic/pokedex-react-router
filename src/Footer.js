import React, { Component } from "react";

export default class Footer extends Component {
  render() {
    return <div>
      <span id="copyright">&copy; 2020 - Poke Nerds LLC</span> 
        </div>
  }
}
