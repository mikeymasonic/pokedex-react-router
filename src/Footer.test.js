import React from 'react';
import Footer from './Footer';
import renderer from "react-test-renderer";
test('renders Footer.js correctly', () => {
  const tree = renderer
  .create(<Footer />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});