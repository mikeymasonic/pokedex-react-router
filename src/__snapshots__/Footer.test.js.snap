// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer.js correctly 1`] = `
<div>
  <span
    id="copyright"
  >
    Â© 2020 - Poke Nerds LLC
  </span>
</div>
`;
